app/src/main/res/layout/state_fragment.xml

```diff
@@ -180,4 +181,4 @@
         android:layout_height="124dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
added